lib/matplotlib/lines.py

```diff
@@ -592,6 +599,59 @@ def get_markevery(self):
         """return the markevery setting"""
         return self._markevery
 
+    def set_downsample(self, downsample):
```

Comment:
I'm down for whatever. I was trying to use the `markevery` option as a guide for how to format the `downsample` option, but that was out of ignorance more than anything else. Just let me know what format you want it in and I'll refactor to that.